src/main/java/uk/gov/hmcts/darts/common/entity/ExternalObjectDirectoryEntity.java

```diff
@@ -93,6 +93,10 @@ public class ExternalObjectDirectoryEntity extends CreatedModifiedBaseEntity imp
     @Column(name = "osr_uuid")
     private Long osrUuid;
 
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Its one-to-one relation. FYI, there is no foreign key relations between the tables.

Reply:
Updated :)